app/[locale]/developers/page.tsx

```diff
@@ -63,6 +65,17 @@ const DevelopersPage = async ({
     namespace: "common",
   })
 
+  const quickstarts = [
+    {
+      description: t("page-developers-quickstart-1-description"),
+      command: "npx create-eth@latest",
```

Comment:
as a developer, imo we should add context to these commands. At least a link to the doc for that command like https://github.com/scaffold-eth/create-eth or maybe better https://docs.scaffoldeth.io/

Reply:
ok lets have links to solidity and scaffold docs, but instead of buttons have it as hyperlink